src/theme/index.ts

```diff
@@ -0,0 +1,34 @@
+import colors from './colors'
+
+export const theme = {
+    breakpoints: [576, 768, 991, 1220],
+    fontSizes: [12, 14, 16, 20, 24, 32],
+    fontWeights: [100, 200, 300, 400, 500, 600, 700, 800, 900],
+    fontFamily: ["'Fira Sans',sans-serif", "'Poppins', sans-serif"],
+    lineHeights: {
+        normal: 1,
+        normalText: 1.5,
+        title: 2,
+        text: 2,
+      },
+      letterSpacings: {
+        normal: 'normal',
+        tracked: '0.1em',
+        tight: '-0.05em',
+        mega: '0.25em',
+      },
+      borders: [
+        0,
+        '1px solid',
+        '2px solid',
+        '3px solid',
+        '4px solid',
+        '5px solid',
+        '6px solid',
+      ],
+      radius: [3, 4, 5, 10, 20, 30, 60, 120, '50%'],
```

Comment:
Do you know if this works with styled-system? I saw in theme docs that they have field called `radii`, but maybe there is also some alias for this? 😄

Reply:
Just checked, should be radii, I'll fix it.